### PR TITLE
Compute columns unique values

### PR DESCRIPTION
**Pull request by rhuille**
Please, read commit by commit.

### CODE REVIEW

File: src/lib/dataset/helpers.ts [hunk 1]
```diff
@@ -92,3 +92,70 @@ export function addLocalUniquesToDataset(dataset: DataSet) {
...
+ * The output will be:
+ * ['City', {
+ *   Paris:  {value: 'Paris', count: 10},
+ *   Lyon:  {value: 'Lyon', count: 1},
+ *   Marseille:  {value: 'Marseille', count: 2},
+ * }]
+ */
+export function _prepareColumnStats(uniqueStatsDataset: DataSet): [string, ColumnValueStat[]] {
+  const columns: string[] = uniqueStatsDataset.headers.map(e => e.name);
+  const indexOfCount: number = columns.indexOf('count'); // equal to 1 in the example given above
```
Comment (adimascio) [inline src/lib/dataset/helpers.ts:119]: Are column names hardcoded? What if the original column name was `count`?

Besides, why doing `1 - indexOfCount` to compute `indexOfColumn` ? If something, shouldn't it rather be `indexOfCount - 1`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ### Added
 
 - New component `ListUniqueValues` which will be used to display column's unique values.
+- When the dataset is updated, compute unique values for each columns using available rows and insert them in dataset's header
```
Comment (adimascio) [inline CHANGELOG.md:12]:
```suggestion
- When the dataset is updated, compute unique values for each column using available rows and insert them in dataset's header
```

File: src/lib/dataset/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+ * }
+ * The output will be:
+ * ['City', {
+ *   Paris:  {value: 'Paris', count: 10},
+ *   Lyon:  {value: 'Lyon', count: 1},
+ *   Marseille:  {value: 'Marseille', count: 2},
+ * }]
+ */
+export function _prepareColumnStats(uniqueStatsDataset: DataSet): [string, ColumnValueStat[]] {
+  const columns: string[] = uniqueStatsDataset.headers.map(e => e.name);
```
Comment (adimascio) [inline src/lib/dataset/helpers.ts:118]:
```suggestion
  const columns = datasetColumnNames(uniqueStatsDataset);
```

File: src/store/backend-plugin.ts [hunk 4]
```diff
@@ -7,17 +7,19 @@
...
 import { PipelineInterpolator } from '@/lib/templating';
-import { VQB_MODULE_NAME, VQBnamespace } from '@/store';
-import { activePipeline } from '@/store/state';
 
 import { StateMutation } from './mutations';
+import { activePipeline } from './state';
+import { VQB_MODULE_NAME, VQBnamespace } from './utils';
+
+// FIXME: type VQBStoreLike = Store<any> | ActionContext<VQBState, any>;
+type VQBStoreLike = any;
```
Comment (adimascio) [inline src/store/backend-plugin.ts:22]: Can it be fixed?

**Comment by rhuille** [comment]
Please find the whole work on column unique values here: https://github.com/ToucanToco/weaverbird/pull/504

**Pull request closed without merging**